ci/verify_bundle.py

```diff
@@ -65,52 +58,18 @@ def _get_net_id(bundle: str):
     return "network_def"
 
 
-def _check_missing_keys(file_name: str, bundle_path: str, keys_list: List):
-    config = ConfigParser.load_config_file(os.path.join(bundle_path, "configs", file_name))
-    missing_keys = []
-    for key in keys_list:
-        if key not in config:
-            missing_keys.append(key)
-
-    if len(missing_keys) > 0:
-        raise ValueError(f"missing key(s): {str(missing_keys)} in {file_name}.")
-
-    return config
-
-
-def _check_main_section_necessary_key(necessary_key: str, config: dict, main_section: str = "train"):
-    # `necessary_key` must be in `main_section`
-    if necessary_key not in config[main_section]:
-        raise ValueError(f"'{necessary_key}' is not existing in '{main_section}'.")
-
-
-def _check_sub_section_necessary_key(
-    necessary_key: str, config: dict, main_section: str = "train", sub_section: str = "trainer"
-):
-    # `necessary_key` must be in `sub_section`
-    if necessary_key not in config[main_section][sub_section]:
-        raise ValueError(f"'{necessary_key}' is not existing in '{main_section}#{sub_section}'.")
-
-
-def _check_main_section_optional_key(
-    arg_name: str, necessary_key: str, config: dict, main_section: str = "train", sub_section: str = "trainer"
-):
-    # if `arg_name` is in `sub_section`, its value must be `necessary_key`
-    if arg_name in config[main_section][sub_section]:
-        if necessary_key not in config[main_section]:
-            actual_key = str(config[main_section][sub_section][arg_name]).split("#")[-1]
-            raise ValueError(f"'{main_section}' should have '{necessary_key}', got '{actual_key}'.")
-
+def _produce_fake_weights(config_file: str, meta_file: str, bundle: str, device: str):
```

Comment:
This function is not used in added property checks. I implement it in case we need it for further tests. cc @MingxueG